integration_tests/test_snap.py

```diff
@@ -149,6 +149,14 @@ def test_snap_from_snapcraft_init(self):
 
         self.run_snapcraft('snap')
 
+    def test_implicit_command_with_arch(self):
+        self.assertThat('snapcraft.yaml', Not(FileExists()))
+        self.run_snapcraft('init')
+        self.assertThat(os.path.join('snap', 'snapcraft.yaml'), FileExists())
```

Comment:
No need for these two assertions, IMO. We start in a temp home, so the directory will be empty. And if init fails, the following command would fail too.